src-tauri/src/mocks/mock_generator.rs

```diff
@@ -0,0 +1,74 @@
+use crate::aux_data_structures::neighbor_info::{Neighbor, NeighborInfo};
+use rand::seq::SliceRandom;
+/*
+Append two GPS decimal points to beginning of randomly generated latitude and longitude to simulate a moving network
+within the Hanover area.
+GPS precision is as follows:
+1 decimal point - 11.1 km
+2 decimal points - 1.11 km
+3 decimal points - 110 meters
+4 decimal points - 11 meters
+See https://gis.stackexchange.com/questions/8650/measuring-accuracy-of-latitude-and-longitude/8674#8674
+*/
+
+pub const HANOVER_LON_PREFIX: f64 = 43.70;
+pub const HANOVER_LAT_PREFIX: f64 = 72.28;
+
+pub fn mock_generator(num_nodes: i32, percent_connected: f64) -> Vec<NeighborInfo> {
+    let mut neighborinfo_vec = Vec::new();
+    for node_id in 0..num_nodes {
+        let mut new_neighbor = NeighborInfo {
+            selfnode: Neighbor {
+                // Assign sequential ids
+                id: node_id as u32,
```

Comment:
I think best to just have node id and snr here. And then we can have a lookup table for id --> node_fts (gps etc.)

Reply:
We would also need speed and direction of each node

Reply:
I think the issue is that speed and direction are typically super inaccurate on the devices, not sure how much we would want to weight this, if at all

Reply:
I believe the speed is determined by taking the difference of current and prior location. On average that's still a useful information. I am considering of using direction for something else (might or might not be useful, I will need to test, if it's not too much trouble of course).

Reply:
Update: We are including the node id and a sent timestamp, and an id, received timestamp, and snr for each edge. The node ids will be used to get the position info that we already maintain (which includes latitude, longitude, speed, and heading). 

So that info will be available as long as we can retrieve it using node ids.